sql/008-release.sql

```diff
@@ -231,8 +277,11 @@ WITH all_document_types AS (
 )
 SELECT
     id,
-    jsonb_object_agg( coalesce(documentType, ''), documentType_count) total_documentType_counts,
-    sum( documentType_count) total_documents
+    jsonb_object_agg (
```

Comment:
Think only change in this block (and the other blocks like it) is SQL layout? Is this intentional? To me the new layout is way harder to read than the old layout but I don't think we have any guides for this kind of thing so I won't block over this

Reply:
@odscjames this is the output from the command specified in https://kingfisher-views.readthedocs.io/en/latest/develop/sql.html#format-sql-files , which I guess is a bit like a style guide, but expressed as a command!